types/sanctuary/index.d.ts

```diff
@@ -1,17 +1,18 @@
-// Type definitions for sanctuary 0.14
+// Type definitions for sanctuary 2.0
 // Project: https://github.com/sanctuary-js/sanctuary#readme
 // Definitions by: David Chambers <https://github.com/davidchambers>
 //                 Juan J. Jimenez-Anca <https://github.com/cortopy>
 //                 Ken Aguilar <https://github.com/piq9117>
+//                 Michał Kaczanowicz <https://github.com/vicrac>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare var S: Sanctuary.Environment;
 export = S;
 export as namespace S;
 
-type Nullable<A> = A | null;
+type TK = any;  // temporary type used in unfinished type definitions
```

Comment:
@vicrac and @edgarjrg, if you would like to contribute please choose any definition that references `TK` and update it to match v2.0.0. :)

Reply:
Should i do that in another PR ? or how could i commit from here?

Reply:
@vicrac could grant you write access to his fork, then the three of us could work on this pull request together. :)

Reply:
@edgarjrg I will grant you permissions to my repo :)